packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -378,6 +378,47 @@ $btn-sizeXSmall-lineHeight      : 1;
     }
 }
 
+.o-btn--icon.o-btn--sizeLarge {
+    width: 56px;
```

Comment:
I'd probably pull out these value as variables and define them at the top of the file, perhaps:

`$btn--sizeLarge--icon-btnLength`
&
`$btn--sizeLarge--icon-iconSize`

and similar for the other sizes.

Just makes them easier to update in the future if they ever change.